susemanager-utils/susemanager-sls/salt/bootstrap/init.sls

```diff
@@ -74,11 +74,14 @@ no_ssh_push_key_authorized:
     {%- set os_base = 'oracle' %}
   {%- endif %}
   #end of expections
+{%- elif grains['os_family'] == 'openEuler' %}
+  {%- set os_base = grains['os'] %} 
+  {% set osrelease = grains['osrelease_info'][0] %}
 {%- endif %}
 
-{% set bootstrap_repo_url = 'https://' ~ salt['pillar.get']('mgr_server') ~ '/pub/repositories/' ~ os_base ~ '/' ~ osrelease ~ '/bootstrap/' %}
+{% set bootstrap_repo_url = 'https://' ~ salt['pillar.get']('mgr_server') ~ '/pub/repositories/' ~ grains['os_family'] ~ '/' ~ grains['osrelease'] ~ '/bootstrap/' %}
```

Comment:
Please rollback. Some OS can't use the grains, even of openEuler does :-)

Reply:
Done

Reply:
I don't see the rollback. I still see the grains being used.

Reply:
Fixed.